packages/plugin-team/test/integration/spec/get.js

```diff
@@ -0,0 +1,209 @@
+/**!
+ *
+ * Copyright (c) 2015-2016 Cisco Systems, Inc. See LICENSE file.
+ * @private
+ */
+
+import '../..';
+import '@ciscospark/plugin-wdm';
+
+import {assert} from '@ciscospark/test-helper-chai';
+import CiscoSpark from '@ciscospark/spark-core';
+import {every, find, map} from 'lodash';
+import testUsers from '@ciscospark/test-helper-test-users';
+import uuid from 'uuid';
+
+function ensureGeneral(team, conversations) {
+  assert.isConversation(find(conversations, (conversation) => {
+    return team.generalConversationUuid === conversation.id && conversation.tags.includes(`TEAM`);
+  }));
+}
+
+describe(`plugin-team`, () => {
+
+  let kirk, spock, team0, team1, teamConvo0, teamConvo1;
+
+  before(() => testUsers.create({count: 2})
+    .then((users) => {
+      [kirk, spock] = users;
+
+      kirk.spark = new CiscoSpark({
+        credentials: {
+          authorization: kirk.token
+        },
+        config: {
+          conversation: {
+            keepEncryptedProperties: true
+          }
+        }
+      });
+
+      spock.spark = new CiscoSpark({
+        credentials: {
+          authorization: spock.token
+        },
+        config: {
+          conversation: {
+            keepEncryptedProperties: true
+          }
+        }
+      });
+
+      return Promise.all([
+        kirk.spark.mercury.connect(),
+        spock.spark.mercury.connect()
+      ]);
+    })
+  );
+
+  before(() => {
+    team0 = {
+      displayName: `test-team-0`,
+      summary: `test-team-0-summary`,
+      participants: [
+        kirk,
+        spock
+      ]
+    };
+
+    team1 = {
+      displayName: `test-team-1`,
+      participants: [
+        kirk
+      ]
+    };
+
+    return Promise.all([
+      kirk.spark.team.create(team0),
+      kirk.spark.team.create(team1)
+    ])
+      .then((teams) => {
+        team0 = teams[0];
+        team1 = teams[1];
+
+        const emptyRoom = {
+          displayName: `team-conversation-${uuid.v4()}`,
+          participants: [
+            kirk
+          ]
+        };
+
+        return Promise.all([
+          kirk.spark.team.createConversation(team0, emptyRoom),
+          kirk.spark.team.createConversation(team0, emptyRoom)
+        ]);
+      })
+      .then((conversations) => {
+        [teamConvo0, teamConvo1] = conversations;
+      });
+  });
+
+  after(() => Promise.all([
+    kirk.spark.mercury.disconnect(),
```

Comment:
something to add in the next PR: make sure to check for existence of kirk and spock. if the before() fails becuase CI, the after will add an extra line to the test report (because kirk and spock are undefined).

in other words, change this to 

after(() => Promise.all([
  kirk && kirk.spark.mercury.disconnect(),
  spock && spock.spark.mercury.disconnect()
]);